easybuild/easyconfigs/s/Singularity/Singularity-2.3.1-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'Singularity'
+version = '2.3.1'
+
+homepage = 'http://gmkurtzer.github.io/singularity'
+description = """Singularity is a portable application stack packaging and runtime utility."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+
+source_urls = ['https://github.com/gmkurtzer/singularity/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['d25be31b443969b12179fbdb6db38c24002f06234bf923424bda1d662ad34e71']
+
+builddependencies = [('Autotools', '20150215')]
+
+preconfigopts = './autogen.sh && '
+
+sanity_check_paths = {
+    'files': ['bin/run-singularity', 'bin/singularity', 'etc/singularity/singularity.conf',
+              'libexec/singularity/bin/action-suid'],
+    'dirs': ['etc', 'libexec/singularity'],
+}
+
+# next steps after instalations
```

Comment:
I know you didn't write this @frenchwr but could you fix the spelling of 'installation' here? I'm finding it painful...